gcloud_dataproc/submit.py

```diff
@@ -95,4 +95,7 @@
     """ % locals()
 
 print(command)
-os.system(command)
+return_code = os.system(command)
```

Comment:
I think [`subprocess.check_call`](https://docs.python.org/2/library/subprocess.html#subprocess.check_call) would be preferable for this.